src/test/coinstatsindex_tests.cpp

```diff
@@ -76,10 +76,12 @@ BOOST_FIXTURE_TEST_CASE(coinstatsindex_initial_sync, TestChain100Setup)
 
     BOOST_CHECK(block_index != new_block_index);
 
+    // Make sure coin_stats_index is not used by the scheduler thread after it
+    // has been destroyed.
```

Comment:
In commit "test: fix race in shutdown code in coinstatsindex_initial_sync" (d2395c83c406f96e3c8c3d638684823c7ada493c)

After #26215 is merged, this comment "Make sure coin_stats_index is not used by the scheduler thread after it has been destroyed" will be misleading, because the previous `BlockUntilSyncedToCurrentChain` above will already be waiting for the scheduler thread, and the only purpose of this `SyncWithValidationInterfaceQueue` call will be to silence TSAN errors.

Reply:
Reworded.